egs/callhome_diarization/v1/local/kaldi_io.py

```diff
@@ -0,0 +1,627 @@
+#!/usr/bin/env python
```

Comment:
This is probably not the right place for this file. @HuangZiliAndy, do you know where this should go?

Reply:
What is this needed for?  I think we should either put it somewhere shared, or not use it at all.